hoomd/hpmc/update.py

```diff
@@ -14,388 +14,171 @@
-            weight (float): relative weight of this box move type relative to other box move types. 0 disables this move type.
-
-        Sample the isobaric distribution of box volumes by rescaling the box.
-
-        Note:
-            When an argument is None, the value is left unchanged from its current state.
-
-        Example::
-
-            box_update.volume(delta=0.01)
-            box_update.volume(delta=0.01, weight=2)
-            box_update.volume(delta=0.01, weight=0.15)
-
-        Returns:
-            A :py:class:`dict` with the current values of *delta* and *weight*.
-
-        """
-        self.check_initialization();
-
-        if weight is not None:
-            self.volume_weight = float(weight)
-
-        if delta is not None:
-            self.volume_delta = float(delta)
-
-        self.cpp_updater.volume(self.volume_delta, self.volume_weight);
-        return {'delta': self.volume_delta, 'weight': self.volume_weight};
+            * ``weight`` (float) - relative weight of log(V) box moves relative to other box move types.
+            * ``delta`` (float) - maximum change of **ln(V)** (where V is box area (2D) or volume (3D)).
 
-    def ln_volume(self, delta=None, weight=None):
-        R""" Enable/disable isobaric volume move and set parameters.
+        aspect (dict):
+            Enable/disable isobaric aspect ratio moves and set parameters. The dictionary
+            has the following keys:
 
-        Args:
-            delta (float): maximum change of **ln(V)** (where V is box area (2D) or volume (3D)).
-            weight (float): relative weight of this box move type relative to other box move types. 0 disables this move type.
-
-        Sample the isobaric distribution of box volumes by rescaling the box.
-
-        Note:
-            When an argument is None, the value is left unchanged from its current state.
-
-        Example::
+            * ``weight`` (float) - relative weight of aspect box moves relative to other box move types.
+            * ``delta`` (float) - maximum relative change of box aspect ratio.
 
-            box_update.ln_volume(delta=0.001)
-            box_update.ln_volume(delta=0.001, weight=2)
-            box_update.ln_volume(delta=0.001, weight=0.15)
-
-        Returns:
-            A :py:class:`dict` with the current values of *delta* and *weight*.
-
-        """
-        self.check_initialization();
+        length (dict):
+            Enable/disable isobaric box length moves and set parameters (change box lengths independently).
+            The dictionary has the following keys:
 
-        if weight is not None:
-            self.ln_volume_weight = float(weight)
+            * ``weight`` (float) - maximum change of the box thickness for each pair of parallel planes
+                     connected by the corresponding box edges. I.e. maximum change of HOOMD-blue box parameters Lx, Ly, Lz.
+            * ``delta`` (list or tuple) -  maximum change of the box tilt factor xy, xz, yz.
```

Comment:
Looks like this docstring is incorrectly copied.
```suggestion
            * ``delta`` (list or tuple) -  maximum change of the box lengths Lx, Ly, Lz.
```